pyproject.toml

```diff
@@ -11,7 +11,8 @@ description-file = "README.md"
 classifiers = ["License :: OSI Approved :: MIT License"]
 requires-python = ">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*"
 requires = [
-  "pathlib2; python_version < '3.4'"
+  "pathlib2; python_version < '3.4'",
```

Comment:
pathlib2 is quite heavyweight, I'd recommend instead of this import lets just make a small ``_pathlib`` file we can fall back to that uses os.path operations underneath, this probably requires just like 5 methods to implement for now